Lombiq.DataTables/package.json

```diff
@@ -1,15 +1,49 @@
 {
   "private": true,
-  "scripts": {
-    "dotnet-prebuild": "gulp"
+  "// inactive dependencies - currently replaced by static files with custom fixes": {
+    "datatables.net": "1.10.25",
+    "datatables.net-bs4": "3.2.2"
   },
   "dependencies": {
-    "@babel/preset-env": "^7.14.8",
-    "datatables.net": "1.10.25",
-    "datatables.net-bs4": "3.2.2",
     "datatables.net-buttons": "1.7.1",
     "datatables.net-buttons-bs4": "1.7.1",
-    "gulp-babel": "^8.0.0",
     "urijs": "1.19.7"
+  },
+  "scripts": {
+    "build": "npm explore nodejs-extensions -- pnpm build",
+    "clean": "npm explore nodejs-extensions -- pnpm clean",
+    "watch": "npm explore nodejs-extensions -- pnpm watch"
+  },
+  "nodejsExtensions": {
+    "assetsToCopy": [
+      {
+        "sources": [ "Assets/NodeModules/datatables.net/js" ],
```

Comment:
Could it be better to rename this directory to something like "StaticNodeModules" or "UnmanagedNodeModules" to be clear it's not just node_modules in a different location?

Reply:
I'm always open for better names. _UnmanagedNodeModules_ is a good choice.